test/config/utility.h

```diff
@@ -67,6 +69,10 @@ class ConfigHelper {
       bool validate_clusters, envoy::api::v2::RouteAction::ClusterNotFoundResponseCode code,
       envoy::api::v2::VirtualHost::TlsRequirementType type = envoy::api::v2::VirtualHost::NONE);
 
+  // Add an additional route with a direct (non-proxied) response.
+  void addDirectResponse(const std::string& domains, const std::string& prefix,
+                         Http::Code responseCode, const std::string& body);
```

Comment:
Do you think this will be used widely?  If not I'd just suggest doing this as with void addConfigModifier(HttpModifierFunction function); in testRouterDirectResponse